website/pages/blog/product-updates-8.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "CloudQuery Product Updates #8"
+tag: product
+date: 2023/09/28
+description: >-
+  Monthly updates on CloudQuery product and roadmap.
+author: michal
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Our monthly round-up of CloudQuery product and roadmap updates is here!
+
+TLDR:
+
+- Open source: A new SDK, new plugins, performance improvements
+- Premium (closed source): New policies
+
+## SDKs!
+
+ICYMI, we published [Python](https://www.cloudquery.io/blog/announcing-cloudquery-python-sdk) and [JavaScript](https://www.cloudquery.io/blog/announcing-cloudquery-javascript-sdk) SDKs last month and this month, we followed with Java!
```

Comment:
I'd suggest we take out the abbreviations such as ICYMI and TLDR in case readers don't understand.